### DISCUSSION TITLE
ESP32 AudioKit - streams-audiokit-multioutput - Echo

### DISCUSSION THREAD

**Discussion by mrmonteith** [pschatzmann/arduino-audio-tools#1685] [Q&A] [answered]
I ran the streams-audiokit-multioutput and getting a lot of echos.  
I'm using Arduino IDE 2.3.2 with the latest Audio Tools and all.
The board is an ESP32 Audio Kit - V2.2 A417
Using simple ear buds.

 I thought maybe the power from the laptop was insufficient.  So I used an external battery pack.  Same behavior.

**Comment by pschatzmann** [comment, marked as answer]
Are you sure that you just don't hear the feed-back from the speaker via the microphones ?

**Reply by mrmonteith** [reply]
I'm using earphone.
